workspaces/liferay-partner-workspace/client-extensions/liferay-partner-etc-spring-boot/src/main/java/com/liferay/partner/QueueListener.java

```diff
@@ -124,6 +155,137 @@ public void listener(
 		}
 	}
 
+	private void _assignUser(
+			String accountName, String contactEmailAddress,
+			String contactRoleName, String externalReferenceCode)
+		throws Exception {
+
+		if (contactRoleName.equals("Partner Manager")) {
```

Comment:
Can you put all these strings into constants? (eg private final String _KORONEIKI_ROLE_PARTNER_MANAGER_NAME, ROLE_PARTNER_MANAGER_NAME, ACCOUNT_ROLE_PARTNER_MANAGER_NAME)

They're used multiple times so it's good to have a single source to be organized and to avoid typos